css/splash.css

```diff
@@ -0,0 +1,106 @@
+.splash {
+    position: relative;
```

Comment:
unused

Reply:
this is being used by the instructions pop up div

Reply:
sounds good